resources/css/app.css

```diff
@@ -1,5 +1,6 @@
 @import 'components/vue-slider';
 @import './theme-variables.css';
+@import './components/autocomplete';
 @import './components/pagination.css';
 
```

Comment:
can you add a bit of consistency here, or all with .css or none of them with .css